### ISSUE TITLE
Variant page - New somatic quick links

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#992]
Maybe do #758 first/same time

Anna B suggested:

* [Cancer hotspots](https://www.cancerhotspots.org/#/home)
* Onco KB (already there but not as quick links)
* Cancer bio (confirm link?)

**Comment by TheMadBug** [comment]
So if you look at how QuickLinks works, we provide certain evidence keys to vc_links.js. When showing quick links, we only pass:
variant coordinate
variant string - as in str(variant)
genome build
canonical c.HGVS

There's a few keys that we have at the allele level that we don't currently pass:
clingen_allele_id,

We don't pass gene symbol, as it's not something directly on the allele, though we could extract it from the canonical c.HGVS

Passing the gene_symbol to quick links should just enable OncoKB. Alternatively maybe we put a list of gene symbol quick links separate from the per variant quick links?

Note that quick links was developed as one of the first things of variant classifications, and have sat theere relatively untouched ever since.

**Comment by davmlaw** [comment]
We now have gene level quick links on the gene info summary part of variant details, and on gene symbol page

They are both hidden behind click to expand things as the list is quite long

**Comment by EmmaTudini** [comment]
Re-opening to see whether this makes it into Shariant test on the next deploy

**Comment by TheMadBug** [comment]
I don't believe any Shariant functionality changed here - but can test the Quick Links to make sure they behaved as before.
The quick links that would appear on the gene symbol page should be hidden in the Shariant view.

**Comment by lhomas** [comment]
Test Run: https://app.testiny.io/VG/testruns/tr/32/tc/188

All test cases passed.

However, there is an issue where there are a number of quicklinks displayed for GRCh37 on the allele that should not be displayed. I discussed with with Dave, who said this is better suited for a new issue rather than a failed test run. I will create a new issue for this (will link here once created).

**Comment by lhomas** [comment]
@EmmaTudini what is the expected/preferred behavior for the "Show Quick Links" button on the allele page? 

Currently, if the annotation for the variant is successful a set of 14 links are displayed (Beacon, cBioPortal, CIViC, ClinGenKB, Clin Trials, Cosmic, GHR, gnomAD, GTEx, Monarch Phen, NCBI, OncoKB, USCS, varsome). If the annotation fails, a set of four links are displayed (Beacon, gnomAD, USCS, varsome). (See images attached below)

However, if you click through to the variant page for either genome build from the allele page, only the set of four links is displayed, regardless of links displayed on the allele page.

14 links:
<img width="442" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUUwQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fe5c5eb2a63f77c9c398527c1802a09aab69eeb6/image.png" alt="image.png" />

4 links:
<img width="441" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUkwQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e23a4f77301a8a0d10b6534246daa239af26f8f5/image.png" alt="image.png" />

**Comment by EmmaTudini** [comment]
Would be good to get annotation working again, then I assume the problem will be fixed? 
James can set test to not shut down if you need to finish the annotations? Just need to make sure we send test back to usual sleep after it's done.
